### PR TITLE
[openui5] Update definition files for OpenUI5 1.92

### PR DESCRIPTION
**Pull request by akudev**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).  → as detailed in the [previous pull request for 1.90](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53680), some are unavoidable at this point
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: → no specific URL about the changes, it's just a new release: https://openui5.hana.ondemand.com/1.92.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/openui5/tslint.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
 		"no-single-declare-module": false,    // these modules do exist and we want TypeScript to know about them
 		"interface-over-type-literal": false, // this rule is considered debatable (see https://github.com/palantir/tslint/issues/3248)
 
-		// the following rules have already been fixed in the "main" branch of OpenUI5, but did not make the 1.92 release
-		"no-redundant-undefined": false,
+		// the following rules have already been fixed in the "main" branch of OpenUI5, but the fixes did not make the 1.92 release
```
Comment (codeworrior) [inline types/openui5/tslint.json:20]: ...it into...

Comment (akudev) [reply types/openui5/tslint.json:20]: Ummm... not necessarily.
https://www.google.com/search?q=%22didn%27t+make+the+release%22+site%3Agithub.com

Actually, your version has less hits:
https://www.google.com/search?q=%22didn%27t+make+it+into+the+release%22+site%3Agithub.com

Comment (codeworrior) [reply types/openui5/tslint.json:20]: I'm tempted to argue with ["Andreas is right"](https://www.google.de/search?q=%22Andreas+is+right%22+site%3Agithub.com) (33) vs. ["Frank is right"](https://www.google.de/search?q=%22Frank+is+right%22+site%3Agithub.com) (141), but okay ¯\\_(ツ)_/¯

**Review by codeworrior — APPROVED** [review]

**Comment by akudev** [comment]
Ready to merge

**Pull request merged**